data/frame.go

```diff
@@ -400,18 +399,18 @@ func FrameTestCompareOptions() []cmp.Option {
 			x == y
 	})
 
-	times := cmp.Comparer(func(x, y time.Time) bool {
-		if !x.Equal(y) {
-			// Check that the millisecond precision is the same.
-			// Avoids problems like:
-			// - s"1970-04-14 21:59:59.254740991 -0800 PST",
-			// + s"1970-04-14 21:59:59.254 -0800 PST",
-			xMS := x.UnixNano() / int64(time.Millisecond)
-			yMS := y.UnixNano() / int64(time.Millisecond)
-			return xMS == yMS
-		}
-		return true
-	})
+	// times := cmp.Comparer(func(x, y time.Time) bool {
```

Comment:
1. Remove commented code
2. Make another function that exports this option specifically, so it can be added to the df test compare options are not
3. Note it as a breaking change (people's tests may fail that previous passed)